config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml

```diff
@@ -72,3 +72,43 @@ periodics:
       - "--timeout=450m"
       securityContext:
         privileged: true
+
+- interval: 6h
+  name: ci-kubernetes-e2e-win-staging
+  labels:
+    preset-service-account: "true"
+    preset-azure-acsengine: "true"
+    preset-k8s-ssh: "true"
+    preset-dind-enabled: "true"
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20190214-f4092ae69-master
+      args:
+      - "--job=$(JOB_NAME)"
+      - "--root=/go/src"
+      - "--repo=k8s.io/kubernetes=master"
+      - "--upload=gs://kubernetes-jenkins/logs/"
+      - "--timeout=480"
+      - "--scenario=kubernetes_e2e"
+      - --
+      - "--test=true"
+      - "--up=true"
+      - "--down=true"
+      - "--deployment=acsengine"
+      - "--provider=skeleton"
+      - "--build=bazel"
+      - "--acsengine-location=westus2"
+      - "--acsengine-admin-username=azureuser"
+      - "--acsengine-admin-password=AdminPassw0rd"
```

Comment:
why is the password in plain text here when we have the environment variable $azure_credentials available? this is duplicated once more in this file.

Reply:
that's actually different. $azure_credentials has what's needed to create the VMs.

This username/password can be used to log into the agent VMs. Those are behind a firewall and not accessible. The next step I'm working on is to switch this off password auth and create single-use SSH keypairs for the agent nodes.

Reply:
@PatrickLang Yes, pretty much like Patrick said: $azure_credentials is used to store info about Azure subscription etc. There is no point in having test-infra folks create and maintain a secret for this. The clusters are generated for a job and destroyed afterwards, the Windows nodes are not publicly accessible so there is not much one can do with this password.